Automattic/wp-calypso
# Plugin Post-Purchase: Add metrics for buttons and page view

#### Proposed Changes
Add track events for:
* "Manage Plugin" button click - `'calypso_plugin_thank_you_manage_plugin_click'`
* "Plugin guide" button click -  `'calypso_plugin_thank_you_plugin_guide_click'`
* "Plugin support" button click - `'calypso_plugin_thank_you_plugin_support_click'`
* "Explore plugins" button click - `'calypso_plugin_thank_you_explore_plugins_click'`
* "Ask a question" button click - `'calypso_plugin_thank_you_ask_question_click'`
* Post-Purchase page view - path -> `"/marketplace/thank-you/:productSlug/:site"`

#### Testing Instructions
* Go to Plugins Post-purchase page of a installed plugin: `/marketplace/thank-you/{plugin_slug}/{site}`
* Click on the buttons described above and check if the events are being tracked properly

#### Pre-merge Checklist





Fixes #70895
